src/public-modules/Review/sagas.js

```diff
@@ -0,0 +1,32 @@
+import request from 'utils/request';
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import { actionTypes, actions } from 'public-modules/Review';
+import { LIMIT } from './constants';
+
+const { POST_REVIEW } = actionTypes;
+const { postReviewSuccess, postReviewFail } = actions;
+
+export function* postNewReview(action) {
+  const { bountyId, fulfillmentId, rating, review } = action;
+
+  // for some reason, this only works when I pass a copy of review
+  const _review = review;
```

Comment:
Looks like there are 2 variables created called `review`, one taken from the `action` and one created from `yield call()` - maybe that could have somethign to do with it.

Reply:
that was the problem hahaha what a dumb mistake

Reply:
Sweet! I've lost count of the number of silly things rubber ducking has helped me with over the years 😂